lib/algos/list/delimiterVersions.js

```diff
@@ -0,0 +1,87 @@
+'use strict'; // eslint-disable-line strict
+
+const Delimiter = require('./delimiter').Delimiter;
+const VSUtils = require('../../versioning/utils').VersioningUtils;
+
+/**
+ * Extended delimiter class for versioning.
+ */
+class DelimiterVersions extends Delimiter {
+    /**
+     *  Constructor of the extension
+     *  Init and check parameters
+     *  @param {Object} parameters - parameters sent to DBD
+     *  @param {RequestLogger} logger - werelogs request logger
+     *  @param {object} latestVersions - latest versions of some keys
```

Comment:
From metadata, as an extension cannot do listing by itself and listing is required to know the latest versions of a collection of objects (listing the master keys).